ph5/clients/ph5availability.py

```diff
@@ -756,18 +756,30 @@ def get_availability(self, station='*', location='*',
                             das_info[ph5_das] = {}
                         k = (channum,  ph5_start_epoch,
                              ph5_stop_epoch, ph5_sample_rate)
-                        if k in das_info[ph5_das].keys():
-                            Das_t = das_info[ph5_das][key]
-                        else:
-                            Das_t = self.ph5.query_das_t(
-                                ph5_das,
-                                chan=channum,
-                                start_epoch=ph5_start_epoch,
-                                stop_epoch=ph5_stop_epoch,
-                                sample_rate=ph5_sample_rate,
-                                sample_rate_multiplier=ph5_multiplier,
-                                check_samplerate=False)
-                            das_info[ph5_das][key] = Das_t
```

Comment:
Also here `das_info[ph5_das][key] = Das_t`.
Then you wont get KeyError anymore.